src/test_queue.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+"""Tests for queue.py"""
+
+
+import pytest
+
+
+from queue import Queue
+
+
+INIT_TABLE = [
+    (1, 1, 1, 1),
+    ([1, 2, 3], 3, 1, 3)
+]
+
+
+def test_init_empty():
+    queue1 = Queue()
+    assert queue1._doublyLL.head_node is None
+    assert len(queue1._doublyLL) == 0
+
+
+@pytest.mark.parametrize('iterable, head, tail, length', INIT_TABLE)
+def test_init(iterable, head, tail, length):
+    queue1 = Queue(iterable)
+    assert queue1._doublyLL.head_node.value == head
+    assert queue1._doublyLL.tail_node.value == tail
+    assert len(queue1._doublyLL) == length
+
+
+def test_enqueue(q0, q4):
+    q0.enqueue(1)
+    q4.enqueue(2)
+    assert q0._doublyLL.head_node.value == 1
```

Comment:
You should be testing your fixtures independently, not with two assert statements in the same test. I can see you are trying to reduce repeated lines of code, and I like that you are aiming for that.